python/orca/src/bigdl/orca/data/shard.py

```diff
@@ -186,6 +186,21 @@ def __init__(self,
         self.type = {}
         if class_name:
             self.type['class_name'] = class_name
+    
+    @classmethod
+    def create(cls,
+               rdd: Union["PipelinedRDD", "RDD"],
+               class_name: str = None) -> "SparkXShards":
+        return cls(rdd, False, class_name)
+    
+    def lazy(self) -> "LazySparkXShards":
+        """
+        Making the current SparkXShards won't affect the current cached rdd.
```

Comment:
Creating `LazySparkXShards` doe not change the existing behavior of the current `SparkXShards`; that is, it will still be uncached unless the user explicitly calls `cache()`

Reply:
should be `that is, it will still be cached unless the user explicitly calls uncache()`??

Updated the doc, please take a further look.